workspaces/adoption-insights/packages/app/src/App.tsx

```diff
@@ -72,7 +74,22 @@ const app = createApp({
     });
   },
   components: {
-    SignInPage: props => <SignInPage {...props} auto providers={['guest']} />,
+    SignInPage: (props: any) => (
+      <SignInPage
+        {...props}
+        align="center"
+        auto
+        providers={[
+          'guest',
+          {
+            id: 'github-auth-provider',
+            title: 'GitHub',
+            message: 'Sign in using GitHub',
+            apiRef: githubAuthApiRef,
```

Comment:
Sign in using Github is showing up in the UI, so can you also add its backend module in packages/backend/src/index.ts and import in package.json?

```
backend.add(import('@backstage/plugin-auth-backend-module-github-provider'));

```